index.html

```diff
@@ -0,0 +1,108 @@
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,900;1,100&display=swap" rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <nav>
+            <div class="nav-bar">
+                <a href="index.html"><img src="./images/Ccsummit2015_logo.png"></a>
+                <ul class="nav-menu">
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="#">Program</a></li>
+                    <li><a href="#">Join</a></li>
+                    <li><a href="#">Sponsor</a></li>
+                    <li><a href="#">News</a></li>
+                </ul>
+                <button class="campaign"><a href="#">Campaign</a></button>
+            </div>
+            <div class="hamburguer">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    <section class="intro">
+        <div class="title">
+            <img src="./images/title.png">
+        </div>
+        <div class="intro-text"> 
+            <h1>A joyful creation believing in the value of openess and sharing, creating a positive change with people
+                over 80 countries is taking place in October, in Korea.</h1>
+        </div>
+        <div class="date-info">
+            <p>2023.05.11(THU) ~ 12(FRI)</p>
+        </div>
+        <div class="location">
+            <p>@RestCasaGrande, Miranda - Venezuela</p>
+        </div>
+    </section>
+    <section class="main">
+    <h2>Main Program</h2>
+    <span class="main-span"></span>  
+    <div class="items">
+        <div class="item">
+        <img src="./images/program_icon_02.png">
+        <h3>Lecture</h3>
+        <p>Small introduction about the meal program and the different menus that will be available on the day of the event</p>
+        </div>   
+        <div class="item">
+        <img src="./images/program_icon_03.png">
+        <h3>Exhibition</h3>
+        <p>Here you will see our best chef working and explaining how each plate of food is made.</p>
+        </div>   
+        <div class="item">
+        <img src="./images/program_icon_04.png">
+        <h3>Tasting</h3>
+        <p>It's time to try the delicacies prepared by our team</p>
+        </div>   
+        <div class="item">
+        <img src="./images/program_icon_05.png">
+        <h3>Feedback</h3>
+        <p>Here we will know your opinion regarding each dish served today</p>
+        </div>
+        <div class="item">
+        <img src="./images/program_icon_01.png">
+        <h3>Dessert</h3>
+        <p>A light dessert before saying goodbye</p>
+        </div>     
+    </div>
+    <a class="program" href="#">SEE THE HOLE PROGRAM</a>
+    </section>
```

Comment:
- [ ] Very nice design :+1: but if your check Behance design in the `mobile version` there is a red colour button in your case, your using a link so kindly fix this issue :slightly_smiling_face: check below :point_down: comparison between your design and Behance design :+1:

**Your design**
![Screenshot from 2023-04-20 04-30-37](https://user-images.githubusercontent.com/45226848/233227083-f5d9833b-e13a-46a8-ae30-1fd8a67f6d56.png)

**Behance Design**
![Screenshot from 2023-04-20 04-30-58](https://user-images.githubusercontent.com/45226848/233227094-32a09334-d962-49dc-a0ea-9c4e0d4e1a86.png)